spec/date_ext_spec.lua

```diff
@@ -20,8 +20,17 @@ describe('Date', function()
 			DateExt.readTimestamp('2021-10-17 17:40 <abbr data-tz="-4:00">EDT</abbr>')
 			assert.stub(FormatDateSpy).was.called_with(mw.language, 'U', '20211017 17:40 -4:00')
 
+			DateExt.readTimestamp('2021-10-17 17:40 - <abbr data-tz="-4:00">EDT</abbr>')
```

Comment:
Did you mean this?
```suggestion
			DateExt.readTimestamp('2021-10-17 - 17:40 <abbr data-tz="-4:00">EDT</abbr>')
```

Reply:
no it usually is between date and time

Reply:
exactly, isn't that what you want to test?